.github/workflows/deploy.yml

```diff
@@ -0,0 +1,60 @@
+
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: [ "Build" ]
+    branches:
+      - master
+      - DEV-100_deployment-on-aws
+    types:
+      - completed
```

Comment:
What does this mean? Just curious.

Reply:
if worklows build was completed (success), then ...

Reply:
👍